From 448f2f7902240f7d3de43681d0a6030ae469113a Mon Sep 17 00:00:00 2001
From: David S. Wang <dsw@cloudera.com>
Date: Wed, 27 Mar 2013 17:06:15 -0700
Subject: [PATCH 47/96] HBASE-8067 TestHFileArchiving.testArchiveOnTableDelete sometimes fails

Reason: Debug
Author: Matteo Bertozzi
Ref: CDH-10058
(cherry picked from commit 3c03b82d82e83da91bfae045b700a2029f02df45)
---
 .../hadoop/hbase/backup/TestHFileArchiving.java    |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/src/test/java/org/apache/hadoop/hbase/backup/TestHFileArchiving.java b/src/test/java/org/apache/hadoop/hbase/backup/TestHFileArchiving.java
index 5d27a98..0edf9f8 100644
--- a/src/test/java/org/apache/hadoop/hbase/backup/TestHFileArchiving.java
+++ b/src/test/java/org/apache/hadoop/hbase/backup/TestHFileArchiving.java
@@ -78,6 +78,9 @@ public class TestHFileArchiving {
   public static void setupCluster() throws Exception {
     setupConf(UTIL.getConfiguration());
     UTIL.startMiniCluster();
+
+    // We don't want the cleaner to remove files. The tests do that.
+    UTIL.getMiniHBaseCluster().getMaster().getHFileCleaner().interrupt();
   }
 
   private static void setupConf(Configuration conf) {
-- 
1.7.0.4

